src/odemis/driver/xt_client.py

```diff
@@ -2202,6 +2206,17 @@ def __init__(self, name, role, parent, hfw_nomag, **kwargs):
         # Instantiate the super scanner class with the update thread
         super(MultiBeamScanner, self).__init__(name, role, parent, hfw_nomag, **kwargs)
 
+    @isasync
+    def applyAutoStigmator(self, detector):
```

Comment:
You don't use the detector, so why pass it as an argument?

Reply:
The other autofunctions use the detector as an argument, as did the code you commented out, so I thought it would be more consistent. But indeed if we are not using it and it's not going to be used in the future, I could remove it.